src/interaction.wiki

```diff
@@ -136,14 +138,14 @@ It is also possible to define your own parameter types.
 
 For our program, we would prefer to take one part of the URL path as the parameter
 describing the name of the user.
-I change the definition of my service this way:
+To achieve this we change the definition of the previous service this way:
 
 <<code language="ocaml"|
 let user_service =
-  Eliom_registration.Html5.register_service
-    ~path:["users"]
-    ~get_params:
-       (Eliom_parameter.suffix (Eliom_parameter.string "name"))
+  Eliom_registration.Html.create
+    ~id:(Eliom_service.Path ["users"])
+    ~meth:(Eliom_service.Get (
+      Eliom_parameter.suffix (Eliom_parameter.string "name")))
```

Comment:
Style-wise I would prefer the shorter `Eliom_parameter.(suffix (string "name"))`, given that there is very little danger of accidentally shadowing something.